src/lang/en-US.json

```diff
@@ -244,7 +244,11 @@
   "heading_region": "Region",
   "heading_patch": "Patch",
   
-  "hero_this_month": "Last 30 days",
+  "hero_pro_tab": "Professional",
+  "hero_public_tab": "Public",
+  "hero_pro_heading": "Picks & Bans in Professional Matches",
+  "hero_public_heading": "Picks & Bans in Public Matches",
+  "hero_this_month": "matches last 30 days",
```

Comment:
we should indicate somehow that the public match data is sampled (doesn't include every match).  It's roughly a 10% sample, but if we are showing absolute numbers I think we need to indicate that.